wremnants/combine_theory_helper.py

```diff
@@ -186,20 +154,25 @@ def set_sum_over_axis(identifier, ax_name):
                 binning = orig_binning
 
             if self.resumUnc:
-                pt30_idx = np.argmax(binning > 30)
+            # if False:
+                pt_idx = np.argmax(binning > 25.)
 
                 if helicity:
-                    # Drop the uncertainties for < 30 for sigma_-1
-                    skip_entries.extend([{"helicity" : -1.j, pt_ax : complex(0, x)} for x in binning[:pt30_idx-1]])
+                    # Drop the uncertainties for low pt for sigma_-1 since this is covered by the resummation uncertainties
+                    # FIXME can't currently mix strings and complex numbers
+                    # skip_entries.extend([{"vars" : "helicity_-1_Down", pt_ax : complex(0, x)} for x in binning[:pt_idx]])
+                    # skip_entries.extend([{"vars" : "helicity_-1_Up", pt_ax : complex(0, x)} for x in binning[:pt_idx]])
+                    skip_entries.extend([{"vars" : "helicity_-1_Down", pt_ax : ibin} for ibin in range(pt_idx)])
```

Comment:
Is helicity_-1_Down/Up going into some histogram name? In that case I would use "m" instead of "-", I'd rather avoid special characters other than underscores in object names, even if it might still work.

Reply:
Yes ```helicity_-1``` is going into the histogram name, and this doesn't seem to cause any problems.  I would rather leave it like this, because otherwise there are several different places where the string substitution becomes more complicated.